tests/framework/CodeInterfaceTests/tests

```diff
@@ -165,4 +165,23 @@
    test_interface_only = True
    rel_err = 0.0001
  [../]
+
+ [./PrescientInterfaceTest]
+   type = 'RavenFramework'
+   input = 'test_Prescient_code_interface.xml'
+   csv = 'PrescientInterfaceTest/out_prescient_0.csv PrescientInterfaceTest/out_prescient_1.csv'
+   rel_err = 0.0001
+   required_executable = 'runner.py'
```

Comment:
runner?

Reply:
Nope, it is runner.py (Personally, I think Prescient should change it (or add one named) to something like prescient_runner but currently it is runner.py)